rules.d/53-storage-hardware.rules

```diff
@@ -78,4 +78,14 @@ SUBSYSTEM=="ubi", KERNEL=="ubi*_*", ATTRS{mtd_num}=="*", ENV{ID_UBI_MTD}="%s{mtd
 # by-path
 ENV{DEVTYPE}=="disk", DEVPATH!="*/virtual/*", IMPORT{builtin}="path_id"
 
+LABEL="block_storage_end"
+
+# SCSI tape devices
+SUBSYSTEM!="scsi_tape", GOTO="storage_hardware_end"
+KERNEL!="st*[0-9]|nst*[0-9]", GOTO="storage_hardware_end"
+
+ATTRS{ieee1394_id}=="?*", ENV{ID_SERIAL}="$attr{ENV{ID_SERIAL}", ENV{ID_BUS}="ieee1394"
+SUBSYSTEMS=="usb", ATTRS{serial}=="?*", IMPORT{builtin}="usb_id"
+IMPORT{builtin}="path_id"
```

Comment:
shouldn#t this be split out into a separate file? 54-storage-hardware-tapes.rules, too?